index.d.ts

```diff
@@ -984,9 +984,10 @@ declare namespace dashjs {
                 longFormContentDurationThreshold?: number,
                 stallThreshold?: number,
                 useAppendWindow?: boolean,
-                setStallState?: boolean
-                avoidCurrentTimeRangePruning?: boolean
-                useChangeTypeForTrackSwitch?: boolean
+                setStallState?: boolean,
+                avoidCurrentTimeRangePruning?: boolean,
+                useChangeTypeForTrackSwitch?: boolean,
+                resetSourceBuffersForTrackSwitch?: boolean
```

Comment:
Please add this new parameter to `index.d.ts`

Reply:
It's right there, or am I missing something?